File: app/src/main/java/foodie/app/rubikkube/foodie/apiUtils/ApiUtils.java
package foodie.app.rubikkube.foodie.apiUtils;

import app.wi.lakhanipilgrimage.api.SOService;

public class ApiUtils {


    public static String BASE_URL = "";
    public static SOService getSOService() {
       BASE_URL = "http://34.220.151.44";
        return RetrofitClient.getClient(BASE_URL).create(SOService.class);
    }

}

File: app/src/main/java/foodie/app/rubikkube/foodie/apiUtils/ApiUtils.kt
package foodie.app.rubikkube.foodie.apiUtils

/**
 * An utility object that acts as a Factory for the
 * REST API client.
 *
 * @author arish
 */
object ApiUtils {
//    const val BASE_URL: String = "http://tecra-r850:8000"
    const val BASE_URL: String = "http://18.237.88.70"

    fun getSOService(): SOService? {
        return RetrofitClient.getClient(this.BASE_URL)?.create(SOService::class.java)
    }
}
